gateway-service-definitions/src/main/resources/services/yarnuiv2/3.0.0/service.xml

```diff
@@ -89,6 +89,14 @@
       <rewrite apply="YARNUIV2/yarnuiv2/inbound/config" to="request.url"/>
     </route>
 
+    <route path="/yarnuiv2/proxy/**">
+      <rewrite apply="YARNUIV2/yarnuiv2/outbound/headers/jobhistory/job" to="response.headers"/>
```

Comment:
Hi @risdenk @moresandeep ,

Please let me know what else needs to be done?

Thanks and Regards,
Deepa